uptane-standard.md

```diff
@@ -728,15 +732,15 @@ The Primary SHALL download metadata for all targets and perform a full verificat
 
 The Primary SHALL download and verify images for itself and for all of its associated Secondaries. Images SHALL be verified by checking that the hash of the image file matches the hash specified in the Director's Targets metadata for that image.
 
-There may be several different filenames that all refer to the same image binary, as described in {{metadata_filename_rules}}. If the Primary has received multiple hashes for a given image binary via the Targets role (see {{targets_images_meta}}) then it SHALL verify every hash for this image despite that the image is identified by a single hash as part of its filename.
+There may be several different filenames that all refer to the same image binary, as described in {{metadata_filename_rules}}. If the Primary has received multiple hashes for a given image binary via the Targets role (see {{targets_images_meta}}) then it SHALL verify every hash for this image even if the image is identified by a single hash as part of its filename.
```

Comment:
Slightly confusing to me. Any given filename only ever uses exactly one hash, so this "if" will always be true.

Reply:
Let me try rephrasing. The "despite that" phrasing read rather awkward to me, but you make a good argument against "if."

Reply:
fixed in latest commit